source/dofs/dof_handler_policy.cc

```diff
@@ -271,6 +287,17 @@ namespace internal
           for (unsigned int d = 0; d < dof_handler.get_fe().dofs_per_line; ++d)
             cell->set_dof_index(d, next_free_dof++);
 
+          // dofs of line
+          for (unsigned int d = 0; d < dof_handler.get_fe().dofs_per_line; ++d)
+            cell->set_dof_index(d, next_free_dof++);
```

Comment:
This just duplicates the 3 lines above -- I think you didn't mean to copy-paste this?